cms/templates/molecules.html

```diff
@@ -0,0 +1,105 @@
+{% import 'atoms.html' as atoms %}
+
+{% macro nav_inline_centered(languages) -%}
```

Comment:
can you look at more meaningful names? I'm digging `{% macro banner_with_image(src)` but this isn't making sense.